src/Benchmarks/Benchmarks.csproj

```diff
@@ -1,34 +1,33 @@
-﻿<Project Sdk="Microsoft.NET.Sdk.Web">
-
-  <Import Project="..\..\build\common.props" />
-
+<Project Sdk="Microsoft.NET.Sdk.Web">
   <PropertyGroup>
     <TargetFramework>netcoreapp2.0</TargetFramework>
     <OutputType>Exe</OutputType>
+    <MvcRazorCompileOnPublish>true</MvcRazorCompileOnPublish>
+    <RuntimeFrameworkVersion>2.0.0-*</RuntimeFrameworkVersion>
```

Comment:
I think at the time @davidfowl filed the issue, the versions were hardcoded in the csproj. But now they are using the versions defined in `dependencies.props`, which is what he wanted. #251 had already been fixed (by me actually, but I wasn't aware there was an open issue for it).